src/util/rest/alias/get.js

```diff
@@ -0,0 +1,4 @@
+module.exports = async (call, idx) => call('GET', '', {
+  endpoint: `_cat/aliases/${idx}`
+})
+  .then((result) => (result.body.length === 0 ? null : result.body.map(({ index }) => index)));
```

Comment:
Returning an array right now. Should we just return one index?

Reply:
I think there should be an assert... and then just return that entry